test/UrlHelperTest.php

```diff
@@ -48,7 +47,7 @@ public function testRaisesExceptionOnInvocationIfNoRouteProvidedAndResultIndicat
 
     public function testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided()
     {
-        $this->router->generateUri('foo', [])->willThrow(RouterException::class);
+        $this->router->generateUri('foo', [], [])->willThrow(RouterException::class);
```

Comment:
Does the additional argument need to be added here (and elsewhere)? Is it always passed by the helper now?